### PR TITLE
radicale: update to 3.2.1, fix conf file permissions

### PR DESCRIPTION
**Pull request by fanyx**
### Changes

- Update to 3.2.1
- https://github.com/void-linux/void-packages/issues/38923

#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64-musl
  - armv7l

This superseeds https://github.com/void-linux/void-packages/pull/46357.

### CODE REVIEW

File: srcpkgs/radicale/template [hunk 1]
```diff
@@ -12,17 +12,17 @@ short_desc="Complete calendar and contact storing and manipulating solution"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
 homepage="https://radicale.org"
-changelog="https://raw.githubusercontent.com/Kozea/Radicale/master/CHANGELOG.md"
+#changelog="https://raw.githubusercontent.com/Kozea/Radicale/master/CHANGELOG.md"
```
Comment (classabbyamp) [inline srcpkgs/radicale/template:15]: changelog should point to master not the tag

File: srcpkgs/radicale/files/radicale/log/run [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```
Comment (classabbyamp) [inline srcpkgs/radicale/files/radicale/log/run:1]: log services should be created automatically by xbps-src if the directory doesn't exist in the service in files/

**Review by classabbyamp — COMMENTED** [review]
squash into 1 commit

File: srcpkgs/radicale/template [hunk 3]
```diff
@@ -18,8 +18,7 @@ distfiles="https://github.com/Kozea/Radicale/archive/refs/tags/v${version}.tar.g
 checksum=4247b18004f0779e00392a3fced1d93f720f50b7be30b46d0283f09de6d1e45b
 conf_files="
  /etc/radicale/config
- /etc/radicale/rights
- /etc/sv/radicale/log/run"
```
Comment (fanyx) [inline srcpkgs/radicale/template:21]: should this line stay or get removed?

Comment (classabbyamp) [reply srcpkgs/radicale/template:21]: not sure why it exists, but probably removed

**Pull request merged**